src/styles/FNDRYPriceWidget.module.css

```
@@ -0,0 +1,323 @@
+
+  .statsGrid {
+    grid-template-columns: 1fr;
+  }
+
+  .metaActions {
+    width: 100%;
+    justify-content: space-between;
+  }
+}
+
+@keyframes pulseUp {
```

Stylelint (17.6.0)

[error] Expected keyframe name "pulseUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)